packages/foreman/rubygem-rb-inotify/rubygem-rb-inotify.spec

```diff
@@ -1,9 +1,12 @@
 # Generated from rb-inotify-0.9.7.gem by gem2rpm -*- rpm-spec -*-
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
 %global gem_name rb-inotify
 
-Name: rubygem-%{gem_name}
+Name: %{?scl_prefix}rubygem-%{gem_name}
```

Comment:
Perhaps you can clean the EL6 compatibility macros from this file while you're at it? That also means the `%doc` can become `%license`.